articles/active-directory/develop/single-page-app-quickstart.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Quickstart: Sign in users in single-page apps (SPA) with authorization code"
-description: In this quickstart, learn how a JavaScript single-page application (SPA) can sign in users of personal accounts, work accounts, and school accounts by using the authorization code flow.
+title: "Quickstart: Sign in users in single-page apps (SPA) with authorization code with Proof Key for Code Exchange (PKCE)"
```

Comment:
title: "Quickstart: Sign in users in single-page apps (SPA) by using the authorization code with Proof Key for Code Exchange (PKCE)"